test/discr/lp_discr_test.py

```diff
@@ -273,6 +273,60 @@ def test_element_from_array_2d_shape():
                        [3, 4]])
 
 
+def test_element_from_function_1d():
+
+    space = odl.uniform_discr(-1, 1, 4)
+    points = space.grid.points().squeeze()
+
+    # Without parameter
+    def f(x):
+        return x * 2 + np.maximum(x, 0)
+
+    elem_f = space.element(f)
+    true_elem = [x * 2 + max(x, 0) for x in points]
+    assert all_equal(elem_f, true_elem)
+
+    # With parameter
+    def f(x, **kwargs):
```

Comment:
No need to take **kwargs here, works without. See for example:

```python
def f(x, c=0):
    return x + c
    
def g(x, **kwargs):
    return f(x, **kwargs)
    
print(g(3, c=2))  # prints 5
```

Also works without `c=0`


```python
def f(x, c):
    return x + c
    
def g(x, **kwargs):
    return f(x, **kwargs)
    
print(g(3, c=2))  # prints 5
```